tests/unit/s2n_record_test.c

```diff
@@ -99,7 +99,7 @@ int main(int argc, char **argv)
         int bytes_written;
 
         EXPECT_SUCCESS(s2n_stuffer_wipe(&conn->out));
-        EXPECT_SUCCESS(bytes_written = s2n_record_write(conn, TLS_APPLICATION_DATA, &in));
+        EXPECT_SUCCESS(bytes_written = s2n_record_write(conn, TLS_ALERT, &in));
```

Comment:
Why would this now be ALERT data?

Reply:
Because `TLS_APPLICATION_DATA` would mean we that it must be encrypted prior, alerts don't have that requirement.